### PR TITLE
Extra integration tests for adding a new reward distribution on L2

### PR DESCRIPTION
**Pull request by i-stam**
- Add `TradingRewards` as a new reward distribution entry on L2.

- Tests whether the rewards as distributed as expected via `L2.RewardsDistribution.distributeRewards() `on L2.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ const defaults = {
 	INITIAL_ISSUANCE: w3utils.toWei(`${100e6}`),
 	CROSS_DOMAIN_DEPOSIT_GAS_LIMIT: `${3e6}`,
 	CROSS_DOMAIN_ESCROW_GAS_LIMIT: `${8e6}`,
-	CROSS_DOMAIN_REWARD_GAS_LIMIT: `${3e6}`,
```
Comment (justinjmoses) [inline index.js:172]: why was this change needed?

**Review by justinjmoses — APPROVED** [review]
Just one question

File: index.js (hunk 1 above)
Comment (i-stam) [reply index.js:172]: This is what was causing the tx to fail. I realised it after everything else that I tried failed.
